test/jdk/jdk/incubator/vector/templates/Unit-header.template

```diff
@@ -1500,6 +1500,47 @@ public class $vectorteststype$ extends AbstractVectorTest {
 #end[intOrLong]
     }
 
+    static $type$ COMPRESSBITS_scalar($type$ a, $type$ b) {
+        $type$ prefix_mask, move_mask, temp;
+        a = ($type$) (a & b);
+        $type$ count_mask = ($type$) (~b << 1);
+        $type$ mp, mv, t;
```

Comment:
These are not used?

Reply:
Thanks for pointing this out. Will fix it.